ar/index.html

```diff
@@ -0,0 +1,164 @@
+      }
+    }
+
+    h1، h2 {
+      line-height: 1.2;
+    }
+
+    footer {
+      grid-row-start: 2;
+      grid-row-end: 3;
+      color: #888;
+      font-size: 9px;
+    }
+
+    a {
+      color: #07A;
+    }
+    a:visited {
+      color: #941352;
+    }
+    footer > a {
+      color: #888 !important;
+    }
+
+    blockquote {
+      background: #F9F9F9;
+      border-right: 10px solid #CCC;
+      margin: 1.5em 10px;
+      padding: 0.5em 10px;
+
+      font-family: monospace;
+    }
+    blockquote p {
+      display: inline;
+    }
+
+    @media screen and (min-width: 400px) {
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>!لا تسأل لتسأل، فقط إسأل </h1>
+    <p>
+ : بين الحين والأخر يأتي احدهم في غرف الدردشة قائلا شيئ مشابه لـ    </p>
+    <blockquote>
+      <span class="name">فلان:</span>
+      <p class="message">
+        هل هناك محترفين جافا هنا؟
+      </p>
+    </blockquote>
+    <p>
+        هذه الطريقة سيئة لأسباب عدة. ما يقصده الشخص
+      <em>في الحقيقة </em>هو 
+    </p>
+    <blockquote>
+      <span class="name">فلان:</span>
+      <p class="message">
+ هل هناك اي محترف جافا يقبل مساعدتي في حل المشكلة، مهما كانت، حتى ولو لم تكن لها علاقة ب"جافا" او ان كان هناك اي شخص لا عرف شيئ عن الجافا يستطيع حل مشكلتي؟     </p>
+    </blockquote>
+    <p>
+  هناك عدة اسباب تدجعل الذين لهم خبرة لا يجيبوا عن سؤالك. سؤالك يسأل أكثر مما تظن.
+    </p>
+    <p>
+  انت تسأل الناس لتحملهم المسؤالية. انت تختبر ثقة الناس في قدراتهم. أنت أيضًا تمنع أشخاصًا آخرين من الإجابة بلا ضرورة.
+
+    </p>
+    <p>
+  يمكن ايضا رؤية سؤالك بهذه الطريقة...
+    </p>
+    <blockquote>
+      <span class="name">فلان:</span>
+      <p class="message">
+   لدي سؤال عن  الجافا لكني اشعر بالكسل لأقوم بصياغته على شكل كلمات الا لو كان بالقناة شخص شخص يستطيع حله.
+
+      </p>
+    </blockquote>
+    <p>
+    ...هو مجرد كسل. إن لم تكن راغبا بالقيام بأي عمل لمساعدة نفسك. لماذا نقوم به نحن؟
+    </p>
+    <p>
+      الحل هو عدم السؤال لسؤال آخر، إسأل مباشرة. من غير المرجح ان يجيب شخص ما على "سؤال لسؤال آخر"، لكن وصفك للمشكلة قد يجذب انتباه العديد القادرين عن الاجابة
+
+    </p>
+    <p>
+      لذلك للتلخيص، لا تسأل      
+<em>"هل يوجد اي خبراء جافا هنا؟"</em>،
+      <em>بل اسأل كيف يمكنني حل (المشكلة الفلانية) باستعمال جافا مع ذكر كامل التفاصيل</em>
+    </p>
+    <br>
+    <br>
+    <p>
+      Other similar problems: <a href="http://xyproblem.info/">The XY Problem</a>، <a href="http://nohello.com/">No Hello</a>.
+      Further reading: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>،
+      or if you have more time: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+  </main>
+  <footer>
+    Content mostly stolen from <a href="http://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    <a href="https://github.com/maunium/dontasktoask.com">Source on GitHub</a>
+  </footer>
```

Comment:
You can add translation credits in the footer

Reply:
do I add the telegram username or the github url ?

Reply:
whatever you want